client/components/SocialLogin.vue

```diff
@@ -9,6 +9,12 @@
         <button @click="socialLogin('facebook')" class="social-login-btn facebook">
             <i class="fab fa-facebook"></i>
         </button>
+        <button @click="socialLogin('twitter')" class="social-login-btn twitter">
```

Comment:
Why did you change these to twitter and linkedin? That doesn't match the Youtube content. (I uploaded this repo because people asked for it because of a youtube tutorial series I did)

Reply:
@fritsvt that's a fair point. I am trying to get all the major social logins working:
google, linkedin, facebook, twitter being the main ones. I understand if you don't want to add them though I don't think people will complain with the added functionality. Your call.

Reply:
It would be ideal if we somehow dynamically showed them when the services are active. I didn't realize you kept the others in though but Ill gladly accept these two new services. Will hopefully show people how easy it should be to add new services.

Reply:
I think that's a good call @fritsvt